tests/ops/test_channel_ops.py

```diff
@@ -314,6 +317,369 @@ def circuit(p_0, p_1):
         )
 
 
+class TestPauliError:
+    """Tests for the quantum channel PauliError"""
+
+    OPERATORS = ["X", "XY", "ZX"]
+    WIRES = [[1], [0, 1], [3, 1]]
+    EXPECTED_KS = [
+        [
+            np.sqrt(0.5) * np.eye(4),
+            np.array(
+                [
+                    [0.0, 0.70710678, 0.0, 0.0],
+                    [0.70710678, 0.0, 0.0, 0.0],
+                    [0.0, 0.0, 0.0, 0.70710678],
+                    [0.0, 0.0, 0.70710678, 0.0],
+                ]
+            ),
+        ],
+        [
+            np.sqrt(0.5) * np.eye(4),
+            np.array(
+                [
+                    [0.0 + 0.0j, 0.0 + 0.0j, 0.0 + 0.0j, 0.0 - 0.70710678j],
+                    [0.0 + 0.0j, 0.0 + 0.0j, 0.0 + 0.70710678j, 0.0 + 0.0j],
+                    [0.0 + 0.0j, 0.0 - 0.70710678j, 0.0 + 0.0j, 0.0 + 0.0j],
+                    [0.0 + 0.70710678j, 0.0 + 0.0j, 0.0 + 0.0j, 0.0 + 0.0j],
+                ]
+            ),
+        ],
+        [
+            np.sqrt(0.5) * np.eye(16),
+            np.array(
+                [
+                    [
```

Comment:
Likely these arrays could be made briefer. Perhaps running `black -l 100 tests/ops/test_channel_ops.py` helps?

Reply:
It's already formatted with the mentioned configuration of black. But considering the change from `max_q = max(wirelist)` to `max_q = len(wirelist)` this will get cleaner